src/js/background.js

```diff
@@ -103,14 +103,21 @@
     return func || null;
   };
 
+  const registerToExtTST = async () => sendMsg(EXT_TST, {
+    type: "register-self",
+    name: browser.i18n.getMessage("extensionName"),
+    icons: browser.runtime.getManifest().icons,
```

Comment:
Isn't runtime.getManifest() async?
Maybe it's better to await for icons and sendMsg()
Something like:
```
const registerToExtTST = async () => {
  const {icons} = await runtime.getManifest();
  return sendMsg(EXT_TST, {
    icons,
    ...
  });
};
```

Reply:
The [API document](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getManifest) says that it returns a JSON object synchronously.

Reply:
I didn't know that, thanks.